### PR TITLE
Add Algorithms/shellSort/python

### PR DESCRIPTION
**Pull request by anubhav4sachan**
**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

Closes addition of Algorithms/shellSort/python

**By submitting this pull request I confirm I've read and complied with the
below declarations.**

- [x] I have read the [Contribution guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md) and I am confident that my PR reflects them.
- [x] I have followed the [commit guidelines](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#write-good-commit-messages) for this project.
- [x] My code follows the [standard code structure](https://github.com/NITSkmOS/Algorithms/blob/master/CONTRIBUTING.md#samples-code).
- [x] This pull request has a descriptive title. For example, `{Tag}: Add {Algorithm/DS name} [{Language}]`, not `Update README.md` or `Added new code`.
- [x] This pull request will be closed if I fail to update it even once in a continuous time span of 7 days.
- [x] This pull request shall only be reviewed and merged once all the checks passes. No maintainer or supporter shall be obliged to review it before this condition is met.
- [x] I have mentioned the issue number correctly (with hyperlink) in this pull request description.

After you submit your pull request, **DO NOT** click the 'Update Branch' button.

### CODE REVIEW

File: shellSort/python/shell_sort.py [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    while gap > 0:
+        for i in range (gap, len(arr)):
+            temp = arr[i]
+            j = i
+            while j >= gap and arr[j - gap] > temp:
+                arr[j] = arr[j-gap]
+                j -= gap
+            arr[j] = temp
+        gap/=2  
+        gap = int(gap)
```
Comment (sangamcse) [inline shellSort/python/shell_sort.py]: What does this function return?

Comment (anubhav4sachan) [reply shellSort/python/shell_sort.py]: Please wait. I'll correct it within 15mins.

**Comment by sangamcse** [comment]
You forgot `git commit --amend` part totally.

**Comment by anubhav4sachan** [comment]
I have not made a copy locally and doing it directly through github.

**Comment by sangamcse** [comment]
Then please stop working like that. We don't want to break our repo for your task completion.

**Comment by anubhav4sachan** [comment]
:') What's happening next? Are you accepting the request?

**Comment by sangamcse** [comment]
Thank you for the contribution. We are not accepting these anymore.

**Pull request closed without merging**